MPF.md

```diff
@@ -792,19 +792,71 @@ allowed values (`LOG_USER`, `LOG_DAEMON`, `LOG_LOCAL0`, `LOG_LOCAL1`,
 
 ### mailto
 
-The address that `cf-execd` should email agent output to.
+The address that `cf-execd` should email agent output to. Defaults to `root@$(default:def.domain)`.
+
+This setting can be customized via Augments, for example:
+
+```json
+{
+  "variables": {
+    "default:def.mailto": {
+        "value": "cfengine-maintainers@example.com",
+        "comment": "Email sent from cf-execd when output differs from the prior execution should be delievered to this distribution group for review"
```

Comment:
```suggestion
        "comment": "When output differs from the prior execution cf-execd will deliver ??? to this Email address for review"
```

What is sent exactly? the diff, the new output, the previous output, both?

Reply:
It's the standard  cf-execd email behavior.

If the output differs from the previous run, all output is emailed (up to maxmaillines). So, e.g. if you put a report with $(sys.date) in your policy, your going to get all the output from the agent run each time.

Assuming no other output change, if you report "Hello world" you get an email with all output and no further email until the output changes.